### PR TITLE
Tool: Mention our use of link shortner and how to create hypha shortlinks

### PR DESCRIPTION
**Pull request by patcon**
Per 2019-06-28 infra meeting, we're going ahead with the link shortening service and need to add mention of how members can interact with it and create shortlinks here.

### CODE REVIEW

**Comment by patcon** [comment]
This is waiting on #10 to be merged, as that creates the "Guides" section.

**Comment by patcon** [comment]
How's this look dc or ben?

File: guides.md [hunk 1]
```diff
@@ -69,6 +69,35 @@ members. Skip the steps below at your discretion for low-stakes topics.
   - as needed, send reminders of action items
   - migrate action items into task tracker
 
+## Managing shortlinks
+
+We use a custom shortlink service at `link.hypha.coop`. It helps us:
+- resolve keywords to URLs independantly of our personal workstation setup,
```
Comment (dcwalk) [inline guides.md]: we haven't been using commas and "and" at the end of bullets lists

Comment (dcwalk) [inline guides.md]: I'm not sure I know that this bullet means @patcon --

Comment (dcwalk) [inline guides.md]:
```suggestion
- resolve keywords to URLs independantly of our personal workstation setup
```

we haven't been using trailing punctuation on bullets

File: guides.md [hunk 2]
```diff
@@ -69,6 +69,35 @@ members. Skip the steps below at your discretion for low-stakes topics.
...
+- make commonly used resources quickly and easily accessible,
+- simplify link-sharing in spoken conversations, and
+- align on shorthand keywords for resources.
+
+Instructions on creating and managing shortlinks are available in
+our [`shortlinks` repo](https://github.com/hyphacoop/shortlinks).
+
+## Accessing shortlinks
+
+Shortlinks work on their own, but for even easier access on your own
```
Comment (dcwalk) [inline guides.md]:
```suggestion
Shortlinks work on their own in the address bar, but for even easier access on your own
```

**Comment by dcwalk** [comment]
This is fine, but I think the mention shouldn't only be in guides.

I think we need like a "one page virtual office cheat sheet", which is pretty standard in a lot of offices and kind of references quick and dirty all the things to get up and running.

**Comment by patcon** [comment]
Good point. Planning to reticket. Would it make sense to frame as an onboarding doc (since that's prob the most obvious time someone will care to read about this?)

File: guides.md (hunk 1 above)
Comment (patcon) [reply guides.md]: ### 
```suggestion
- easily resolve keywords to URLs from any computer,
```

File: guides.md [hunk 3]
```diff
@@ -69,6 +69,35 @@ members. Skip the steps below at your discretion for low-stakes topics.
   - as needed, send reminders of action items
   - migrate action items into task tracker
 
+## Managing shortlinks
+
+We use a custom shortlink service at `link.hypha.coop`. It helps us:
+- resolve keywords to URLs independantly of our personal workstation setup
```
Comment (patcon) [inline guides.md]:
```suggestion
- resolve keywords to URLs from any computer,
```

Comment (patcon) [reply guides.md]: Better @dcwalk ?

Comment (patcon) [reply guides.md]: Better @dcwalk ?

**Comment by patcon** [comment]
Reticketed your comment. Merge if good DC

File: guides.md (hunk 3 above)
Comment (patcon) [reply guides.md]: bump? commit and merge at will, and we'll close organizing#74 :)

Comment (patcon) [reply guides.md]: oo wait. one more addition. +1 but don't merge yet.

File: guides.md [hunk 4]
```diff
@@ -69,6 +69,38 @@ members. Skip the steps below at your discretion for low-stakes topics.
...
+- simplify link-sharing in spoken conversations, and
+- align on shorthand keywords for resources.
+
+Instructions on creating and managing shortlinks are available in
+our [`shortlinks` repo](https://github.com/hyphacoop/shortlinks).
+
+## Accessing shortlinks
+
+> **Hint:** You can use a URL hash to deep-link into an expanded shortlink.
+> Example:&nbsp;http://link.hypha.coop/inventory#MailCow
```
Comment (patcon) [inline guides.md]: Added this. Look ok?

**Pull request merged**